src/Suggestion.js

```diff
@@ -95,7 +102,12 @@ const styled = defaultStyle(
   {
     cursor: 'pointer',
   },
-  (props) => ({ '&focused': props.focused })
+  (props) => ({
+    '&focused': props.focused,
+    '&disabled': Boolean(
+      typeof props.suggestion === 'string' ? false : props.suggestion.disabled
```

Comment:
Why do you check here if 'suggestion' is string before accessing `disabled` properly, but on a line 74, you don't?

Reply:
Good catch @crecotun, thanks - I've added 47b3c68 to be more defensive here